README.md

```diff
@@ -0,0 +1,23 @@
+# Percival.jl - An augmented Lagrangian solver
+
+[![Build Status](https://travis-ci.org/JuliaSmoothOptimizers/Percival.jl.svg?branch=master)](https://travis-ci.org/JuliaSmoothOptimizers/Percival.jl)
+[![Build status](https://ci.appveyor.com/api/projects/status/kp1o6ejuu6kgskvp/branch/master?svg=true)](https://ci.appveyor.com/project/dpo/percival-jl/branch/master)
+[![Build Status](https://api.cirrus-ci.com/github/JuliaSmoothOptimizers/Krylov.jl.svg)](https://cirrus-ci.com/github/JuliaSmoothOptimizers/Krylov.jl)
```

Comment:
This says Krylov.jl.

Reply:
Thanks. Helped find this: https://github.com/JuliaSmoothOptimizers/LinearOperators.jl/pull/150